docs/source/markdown/podman-generate-systemd.1.md

```diff
@@ -94,9 +94,108 @@ PIDFile=/run/user/1000/overlay-containers/ccfd5c71a088768774ca7bd05888d55cc28769
 WantedBy=multi-user.target default.target
 ```
 
+Installation of generated systemd unit files.
+
+Podman generated unit files include an `[Install]` section, which carries installation information for the unit. It is used by the enable and disable commands of the systemctl(1) during installation.
+
+Once you have generated the systemd unit file. Copy the generated systemd file to ```/usr/lib/systemd/system``` for installing as a root user and to ```/usr/lib/systemd/user``` for installing it as a non-root user.
+
+Enable the copied one or more units. This will create a set of symlinks, as encoded in the "[Install]" sections of the indicated unit files.
```

Comment:
Delete this line. It doesn't really add information in a how-to context.

Reply:
Agreed. removed.